pkg/config/server_config.go

```diff
@@ -166,6 +171,7 @@ type ServerConfig struct {
 	KVClient            *KVClientConfig `toml:"kv-client" json:"kv-client"`
 	Debug               *DebugConfig    `toml:"debug" json:"debug"`
 	ClusterID           string          `toml:"cluster-id" json:"cluster-id"`
+	MaxMemoryPercentage int             `toml:"max-memory-percentage" json:"max-memory-percentage"`
```

Comment:
Why not use the `uint` type?

Reply:
Not very necessary, we treat both negative value and zero as disabled option.